nncf/common/pruning/pruning_node_selector.py

```diff
@@ -91,15 +92,15 @@ def __init__(self,
     def create_pruning_groups(self, graph: NNCFGraph) -> Clusterization:
         """
         This function groups ALL nodes with pruning types to groups that should be pruned together.
-        1. Create clusters for special ops (eltwises) that should be pruned together
-        2. Create groups of nodes that should be pruned together (taking into account clusters of special ops)
-        3. Add remaining single nodes
-        4. Unite clusters for Conv + Depthwise conv (should be pruned together too)
-        5. Checks for groups (all nodes in group can prune or all group can't be pruned)
+            1. Create clusters for special ops (eltwises) that should be pruned together
+            2. Create groups of nodes that should be pruned together (taking into account clusters of special ops)
+            3. Add remaining single nodes
+            4. Unite clusters for Conv + Depthwise conv (should be pruned together too)
+            5. Checks for groups (all nodes in group can prune or all group can't be pruned)
         Return groups of modules that should be pruned together.
-        :param graph: graph to work with
-        and their initialization parameters as values
-        :return: clusterisation of pruned nodes
+
+        :param graph: Fraph to work with and their initialization parameters as values.
+        :return: Clusterisation of pruned nodes.
```

Comment:
Clusterization (this spelling is used in other places)

Reply:
Thanks! I fixed it.